src/searchReplace.tsx

```diff
@@ -74,6 +87,7 @@ export class SearchReplaceView extends VDomRenderer<SearchReplaceModel> {
             (this.model.searchString = event.target.value)
           }
         />
+        {this.model.isLoading && <Progress />}
```

Comment:
Could you switch to `ProgressRing` to integrate better with JLab pattern?

And could you add style to place the progress ring in the center of the panel?